### DISCUSSION TITLE
正文有英文单词会超过行宽度

### DISCUSSION THREAD

**Discussion by shawchoy** [note286/xduts#159] [Q&A] [answered]
```
\begin{document}
\section{第一节}
\subsection{第一节}
对基于什么什么什么什么模型进行训练和测试时，我们同样使用LIVEC\cite{data_livec}、Koniq10k\cite{data_Koniq}两个数据集，
\end{document}
```
![QQ图片20240119093506](https://github.com/note286/xduts/assets/94467157/a2fa7643-d9bc-455e-9aa9-c2b204fa0eff)
如果从LIVEC、之后换行那这一行后面空缺比较多，
![A ~8{PTP%Q@YR70M8FS8TBY](https://github.com/note286/xduts/assets/94467157/ef1ee8e7-d5a1-42ff-9c7a-0a84984a1987)
怎么能让这一行字均匀分布

**Comment by note286** [comment, marked as answer]
`Koni\-q10k`

**Reply by shawchoy** [reply]
好的谢谢

**Reply by note286** [reply]
一般来说，LaTeX给出的就是最优解了，我相信你不会想要每个汉字之间有较大间距的。

**Reply by shawchoy** [reply]
因为word论文模板里面会自适应地调整间距，我一开始是想要这个效果的，当然只要符合论文格式规范就行

**Reply by note286** [reply]
我觉得你不会想要那个效果的，不要把Microsoft Word的效果当做标准，你可以从自己的审美角度出发。

**Reply by shawchoy** [reply]
好的谢谢
